.github/actions/spell-check/dictionary/dictionary.txt

```diff
@@ -277038,6 +277040,8 @@ open-spacedness
 open-spoken
 open-spokenly
 open-spokenness
+openssh
+open-ssh
```

Comment:
Same with this one, I'd assume that it should always be "openssh"

Reply:
I just want to save it for some URLs which may contains something like "open-ssh". Who knows. If it isn't for the URLs, I'm OK to remove it.